pkg/plugins/gateway/cloudrun/http.go

```diff
@@ -47,24 +47,36 @@ func middleware(ctx *fasthttp.RequestCtx, pool worker.WorkerPool) bool {
 	if err := json.Unmarshal(bodyBytes, &pubsubEvent); err == nil && pubsubEvent.Subscription != "" {
 		// We have an event from pubsub here...
 
+		topicName, ok := pubsubEvent.Message.Attributes["x-nitric-topic"]
+		if !ok {
+			ctx.Error("Could not find topicName for event", 400)
+			return false
+		}
+
```

Comment:
Wasn't part of this change to retrieve the topic name from a query parameter if it's available

Reply:
it was the idea, but it does not work as it changes the signature and breaks the authorizer on the function.
So i at least wanted a better error.

Reply:
Is there an alternative we could use that would not impact the authorizer? (e.g. using path instead?).

Another alternative could be to look at binding using eventarcs instead as well.